services/app-web/src/components/SubmissionSummarySection/ContractDetailsSummarySection/ContractDetailsSummarySection.tsx

```diff
@@ -82,6 +80,8 @@ export const ContractDetailsSummarySection = ({
     const [zippedFilesURL, setZippedFilesURL] = useState<string>('')
     const contractSupportingDocuments = submission.documents.filter(doc => doc.documentCategories.includes('CONTRACT_RELATED' as const))
     const isSubmitted = submission.__typename === 'StateSubmission'
+    console.log("IS SUB", isSubmitted, submission.__typename)
```

Comment:
eventually what will be the logic  on the frontend for `isEditing`?  Is it status is DRAFT | | UNLOCKED ?

Reply:
I suspect so. The trick I took advantage of here is that we only set navigateTo on the reivew and submit page, when we are editing. It’s not even set on submission summary so we are already communicating edit status to the component.